nsv13/code/controllers/subsystem/overmap_mode.dm

```diff
@@ -305,6 +305,7 @@ SUBSYSTEM_DEF(overmap_mode)
 	for(var/datum/overmap_objective/O in mode.objectives)
 		O.ignore_check = TRUE //We no longer care about checking these objective against completeion
 
+	/* This doesn't work and I don't have the time to refactor all of it right now so on the TODO pile it goes!
```

Comment:
weh

Reply:
Look right now the objectives I chose are the only ones supposed to be eligible for selection anyway so yeah we can just figure it out when we have something that *does* use it